src/isar/state_machine/states/monitor.py

```diff
@@ -69,6 +71,23 @@ def _run(self):
             self.state_machine.current_task.status = task_status
 
             if self._task_completed(task_status=self.state_machine.current_task.status):
+                if isinstance(self.state_machine.current_task, InspectionTask):
+                    inspections: Sequence[
+                        Inspection
+                    ] = self.state_machine.robot.get_inspections(
+                        task=self.state_machine.current_task
+                    )
+
+                    mission_metadata: MissionMetadata = (
+                        self.state_machine.current_mission.metadata
+                    )
+                    for inspection in inspections:
```

Comment:
If there are multiple inspections for the current task, should the metadata be the same for all of them?

Reply:
The mission metadata should be the same. This metadata contains no task specific information but information that is common for the entire mission. As such, this metadata will be the same across tasks as well. However, the inspection object itself also has a metadata object which is specific to that individual inspection.

Reply:
Note that having the same metadata across the mission like this is what links the inspections together in SLIMM.